README.md

```diff
@@ -146,6 +146,12 @@ Lastly, each class should only perform one task, do it well, and be unit tested
 * Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
 * If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
 
+[Bug](https://github.com/fossasia/open-event-orga-app/issues/new?template=bug.md)
```

Comment:
Add these to as high as possible on the doc, below the description quote. Write in this form:

Have an issue? Create one using these:

And these list these in form of buttons from shields.io

Reply:
PR updated.
Please review,I hope I have placed the issue buttons correctly
 https://github.com/nikit19/open-event-orga-app/tree/issueTemplate